### DISCUSSION TITLE
Logging in using Google credentials

### DISCUSSION THREAD

**Discussion by lhassell** [widewing/ha-toyota-na#55] [Q&A]
My wife and I both have out Toyota logins linked to our google logins. How can we log in to the integration? I haven't been able to find a way to login, since we don't have logins to the Toyota app.

**Comment by vager88** [comment]
Try signing out of your Toyota App, so you can get to the Login page again.  From the Login page, instead of signing in with your google account, use the email login and be sure to use the same email as your google account. When you get to the password prompt, click on forgot password. The app will send a 2FA code to your email in which after you enter you will be able to set a password for the account.  Now you can log in on the app with Google or with your email/password.  I'm going to assume this is what you need for this integration.

**Reply by kevinhalpin** [reply]
> Try signing out of your Toyota App, so you can get to the Login page again. From the Login page, instead of signing in with your google account, use the email login and be sure to use the same email as your google account. When you get to the password prompt, click on forgot password. The app will send a 2FA code to your email in which after you enter you will be able to set a password for the account. Now you can log in on the app with Google or with your email/password. I'm going to assume this is what you need for this integration.

This worked for me! Thank you!!

**Comment by themacadmin** [comment]
_[edit] I tried it anyway, and Toyota.com returned a "Service is down" error when attempting to set a password._

Hmm.

I may try this, however it defeats the purpose of using an identity provider with the Toyota account.

Doing this creates a password that is stored with Toyota, and therefore makes the account subject to any account compromises that may occur on Toyota's side. I trust an identity provider's account security far more than I trust a car company's.
